README.md

```diff
@@ -56,7 +57,8 @@ This tap:
      "cloud_id": "<cloud-id>",
      "refresh_token": "<refresh-token>",
      "start_date": "<i.e. 2017-12-04T19:19:32Z>",
-     "request_timeout": 300
+     "request_timeout": 300,
+     "groups": "jira-administrators,site-admins,jira-software-users"
```

Comment:
```suggestion
     "groups": "jira-administrators, site-admins, jira-software-users"
```

Reply:
As discussed [above ](https://github.com/singer-io/tap-jira/pull/88#discussion_r881247537) we can not pass 'groups' value with extra spaces.